stdlib/scanf.mli

```diff
@@ -448,7 +448,7 @@ val bscanf : Scanning.in_channel -> ('a, 'b, 'c, 'd) scanner
 (** Scanners may raise the following exceptions when the input cannot be read
     according to the format string:
 
-    - Raise {!Scanf.Scan_failure} if the input does not match the format.
+    - Raise [Scan_failure] if the input does not match the format.
```

Comment:
This reference is fine that can go back to `{!Scanf.Scan_failure}`

Reply:
```suggestion
    - Raise {!Scanf.Scan_failure} if the input does not match the format.
```